crates/pagecache/src/io/periodic.rs

```diff
@@ -0,0 +1,51 @@
+use std::marker::PhantomData;
```

Comment:
Could you move this file to `crates/pagecache/src`? This looks great :)

Reply:
I moved it :)